app/bundles/LeadBundle/Views/List/form.html.php

```diff
@@ -26,7 +26,7 @@
     'timezones' => 'timezone-template',
     'select'    => 'select-template',
     'lists'     => 'leadlist-template',
-    'emails'    => 'lead_email_received-template',
+    'emails'    => 'lead_email_read-template',
```

Comment:
It's used by https://github.com/mautic/mautic/pull/2118/files#diff-fae7433c5a80f1f1a5a92d1f7e49afcaR308